src/store/index.ts

```diff
@@ -260,6 +231,14 @@ const {
         state.linkWidthScale.domain(domain).range([1, 20]);
       }
     },
+
+    createProvenance(state) {
+      state.provenance = initProvenance<State, ProvenanceEventTypes, unknown>(
```

Comment:
This goes somewhat along with my last comment: Since the provenance is for an independent system that reacts to changes in the state (and possibly causes some changes of its own) from an engineering perspective it seems like it should be encapsulated *outside* of the VueX state.

Reply:
If we follow your suggestion from above, we can have this be a method in the provenance "component" and then call it from wherever we bind data to the provenance (I suggested App.vue above).

For now, I can move the initialization out to the provenanceUtils.ts file, if that sounds like a reasonable compromise

Reply:
You can move this function if you want, but it's not necessary right now. If you want to file an issue to re-examine the design later, that would be more than enough to get this PR out the door.

Reply:
Opened #137